docs/kafka/nodejs-kafka/README.adoc

```diff
@@ -161,15 +161,16 @@ endif::[]
 
 . In your IDE, create a new file. Save the file with the name `rhoas.env`, at the root level of the `reactive-example` directory for the cloned repository.
 
-. In the `rhoas.env` file, add the lines shown in the example. These lines set the bootstrap server and client credentials as environment variables to be used by the Node.js application.
+. In the rhoas.env file, set the SASL connection mechanism and the Kafka instance client credentials as shown in the following configuration. Replace the values with your own credential information. {product-kafka} supports the SASL/OAUTHBEARER mechanism for authentication, which is the recommended authentication mechanism to use.
 +
 .Setting environment variables in the rhoas.env file
 [source,subs="+quotes"]
 ----
 KAFKA_HOST=__<bootstrap_server>__
 RHOAS_SERVICE_ACCOUNT_CLIENT_ID=__<client_id>__
 RHOAS_SERVICE_ACCOUNT_CLIENT_SECRET=__<client_secret>__
-KAFKA_SASL_MECHANISM=plain
+KAFKA_SASL_MECHANISM=oauthbearer
+RHOAS_TOKEN_ENDPOINT_URL=https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token
```

Comment:
There is an available attribute for the token endpoint URL. Therefore:

```
.Setting environment variables in the rhoas.env file
[source,subs="+attributes,+quotes"]
----
KAFKA_HOST=__<bootstrap_server>__
RHOAS_SERVICE_ACCOUNT_CLIENT_ID=__<client_id>__
RHOAS_SERVICE_ACCOUNT_CLIENT_SECRET=__<client_secret>__
KAFKA_SASL_MECHANISM=plain
KAFKA_SASL_MECHANISM=oauthbearer
RHOAS_TOKEN_ENDPOINT_URL={sso-token-url}
```

Reply:
As part of the change suggested above, observe that we updated the `subs` element at the head of the code block to also include attributes.